drivers/os/linux/linux.go

```diff
@@ -67,6 +67,13 @@ func (d *driver) Mounts(
 		return nil, goof.New("cannot specify mountPoint and deviceName")
 	}
 
+	// TODO: s3fs doesnt provide mounted bucket in /proc/self/mountinfo
+	// that's why m.Source is just 's3fs'
+	// so, it is workaround to patch Source info
+	// Another option - use moinPoint to filter mounts into
+	// the integration driver logic and remove this patching
+	d.patchMounts(ctx, &mounts)
```

Comment:
Hi @codenrhoden,

Can you please take a look at the suggested `PatchMountInfo` function as well as the implementation of it in this PR and tell me if I'm crazy, but I think we should take the other option @alexey-mr suggests and relocate this to the integration driver [here](https://github.com/cloudscaling/libstorage/blob/95273fdc3263d24c47e244c00db32b34fb55bd5c/drivers/integration/linux/linux.go#L262).

If I'm following all of the code correctly, it appears that the patching/lookup should really be occurring in the `LocalDevices` function of the executor so that the `Libstorage-Localdevices` map is initialized properly when sent to the server and thus the returned volumes have the mappings correctly set. That would mean the correct device name and source should be set when used in the integration driver.

Thoughts?